web/listener/listener.ts

```diff
@@ -0,0 +1,33 @@
+import { UnistylesRuntime } from '../runtime'
+import { UnistyleDependency } from './dependencyList'
+
+export class UnistylesListenerBuilder {
+    private isInitialized = false
+    private listeners = Array.from({ length: Object.keys(UnistyleDependency).length / 2 }, () => new Set<VoidFunction>())
```

Comment:
hmm?

Reply:
I've wanted `Record<UnistyleDependency, Set<VoidFunction>>` but typescript has a problem with numeric enums, so this is a workaround for such structure